kuka_kr16_support/ros2_control/kr16_2.ros2_control.xacro

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="kr16_2_ros2_control">
+  <xacro:macro name="kr16_2_interface" params="name prefix rsi_ip rsi_port">
+
+    <ros2_control name="${name}" type="system">
+      <hardware>
+        <plugin>robot_interface/kuka/RsiInterface</plugin>
+        <param name="rsi_ip">${rsi_ip}</param>
+        <param name="rsi_port">${rsi_port}</param>
+      </hardware>
+
+      <joint name="${prefix}joint_a1">
+        <command_interface name="position">
+          <param name="min">-3.22885911619</param>
+          <param name="max">3.22885911619</param>
+        </command_interface>
```

Comment:
By using relative axis correction, we can effectively control the velocity (relative displacement at each timestep). To support this in the abstraction layer, we need to add a `<command_interface name="velocity">`. 

The default limit to AXIS CORRECTION is 6º per instruction. If that is done at 250Hz (every 4ms at IPO FAST), it will give an angular velocity of 1500º/s which is beyond the physical capabilities of the robot. As an upper bound, let's round it down to 5º per instruction, which at 250Hz can be further rounded down to `20 radians/second` (about 200rpm).